test/unit/api/regular/arithmetic.cpp

```diff
@@ -126,4 +139,16 @@ TTS_CASE_WITH( "Check behavior of arithmetic operators on wide and scalar"
     TTS_EQUAL( (a0 % s), T([&](auto i, auto) { return a0.get(i) % s; }));
     TTS_EQUAL( (s % a1), T([&](auto i, auto) { return s % a1.get(i); }));
   }
+
+  auto cpy = a0; (cpy += s);
+  TTS_EQUAL(cpy, T([&](auto i, auto) { return a0.get(i) + s; }));
+
+  cpy = a0; (cpy -= s);
+  TTS_EQUAL(cpy, T([&](auto i, auto) { return a0.get(i) - s; }));
+
+  cpy = a0; (cpy *= s);
+  TTS_EQUAL(cpy, T([&](auto i, auto) { return a0.get(i) * s; }));
+
+  cpy = a0; (cpy /= s);
+  TTS_ULP_EQUAL(cpy, T([&](auto i, auto) { return a0.get(i) / s; }), 1);
```

Comment:
run clang format on the file. I don't mind your two steatements on one line but it will - it's not worth it.